person.rb

```diff
@@ -7,14 +7,16 @@ def initialize(age, name: 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+
+    @rental = []
     super()
   end
 
   def correct_name
     @name
   end
 
-  attr_reader :id # getter
```

Comment:
* Also here kindly Create the has-many side of Person and Rental (a person has many rentals). as stated in the [project requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md#:~:text=Create%20the%20has%2Dmany%20side%20of%20Person%20and%20Rental%20(a%20person%20has%20many%20rentals).) like i mentioned above, a method for adding rentals should be present so that we can add new rentals into the attribute @rental